### PR TITLE
refactor: add copy script to auto locate node_modules and copy into dist dir

### PR DESCRIPTION
**Pull request by digitalsadhu**
Why?

Before this PR, location of node_modules folder was hard coded in publish commands and such. This is very brittle because we have no direct control over which node_modules folder npm will install into. If there is a clash then npm will place dependencies into the node_modules folder inside the package folder, otherwise it will place it in node_modules at the root level. 

Initially, I tried to do with using Node's own resolve mechanisms but they are limited to what has been defined in the package.json exports field. Some packages don't use these correctly and even if they do, each file has to be individually copied as you can't glob on a whole directory. Because of this, I ended up using the `findup-sync` package to locate the closest `node_modules` folder parent.

Each package then first either uses this script to copy the files needed for upload to a `dist` folder OR uses a rollup build to build into the `dist` folder. Eik then can just always upload whatever is in the `dist` folder.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,5 +22,9 @@
     "packages/fabric-react",
     "packages/lit-element",
     "packages/lit-html"
-  ]
+  ],
+  "devDependencies": {
+    "@eik/cli": "2.0.16",
+    "findup-sync": "5.0.0"
```
Comment (digitalsadhu) [inline package.json:28]: findup-sync added to locate nearest parent node_modules folder

File: packages/fabric-css/package.json [hunk 2]
```diff
@@ -24,7 +25,7 @@
   "eik": {
     "server": "https://assets.finn.no",
     "type": "npm",
-    "files": "./node_modules/@fabric-ds/css/dist"
+    "files": "dist"
```
Comment (digitalsadhu) [inline packages/fabric-css/package.json:28]: The Eik config is almost always simplified to just upload what's in `dist`

File: packages/lit-element/package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "main": "index.js",
   "scripts": {
     "build": "npx rollup -c",
+    "copy": "echo \"No copy command installed\"",
```
Comment (digitalsadhu) [inline packages/lit-element/package.json:7]: in cases where there is a rollup build, we don't need to run copy so we just add a dummy command to be run instead.

File: packages/react/package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
   "main": "react.production.min.js",
   "scripts": {
     "build": "echo \"No build command installed\"",
+    "copy": "../../scripts/copy.js react @esm-bundle/react esm/react.production.min.*",
```
Comment (digitalsadhu) [inline packages/react/package.json:10]: Args:
1. monorepo package folder name eg.react 
2. dependency package name eg. @esm-bundle/react
3. path within the package folder in node_modules, eg. esm/react.production.min.*

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:

**Pull request merged**